src/main/java/com/robinpowered/sdk/model/Identifiable.java

```diff
@@ -0,0 +1,57 @@
+package com.robinpowered.sdk.model;
+
+import com.google.gson.annotations.SerializedName;
+
+import org.joda.time.DateTime;
+
+/**
+ * Interface for identifiers of {@link Device Devices}.
+ */
+public interface Identifiable {
```

Comment:
Yeah I don't have that. It would be nice to use `Identifier` for hte interface maybe, and call the other class `BasicIdentifier` or something. Then you can return `Identifier` in the retrofit service definition and have your deserializer determine which implementation it should be.